x-pack/plugins/security/server/lib/authorization/privileges.js

```diff
@@ -28,3 +28,7 @@ export function buildPrivilegeMap(savedObjectTypes, application, actions) {
     }
   };
 }
+
+export function buildLegacyIndexPrivileges() {
```

Comment:
Since this is a constant, and we use it like a constant when creating the schema, I'm tempted to suggest that we break from the tradition of naming this `buildLegacyIndexPrivileges` and just using `const LEGACY_INDEX_PRIVILEGES`, what do you think?

Reply:
I thought about introducing a constant here, but wrapping it in a function ensures that other code won't attempt to modify the entries in that array, which the `const` won't protect against. This approach ensures that you can only tamper with the instance you explicitly ask for, rather than the singleton that "everyone" uses.